projects/igniteui-angular-extras/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "igniteui-angular-extras",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^9.0.0",
+    "@angular/core": "^9.0.0",
+    "igniteui-angular": "^9.0.6",
```

Comment:
Those should probably be more restrictive like `~9.0.6`, since this will pick up 9.1.x/9.2.x/etc - major release for us and may contain breaking changes. I believe the same applies to the chart versioning

Reply:
All of the peerDependencies in the package.json are with tilde instead of a caret